src/main/java/org/alfresco/rest/core/RestWrapper.java

```diff
@@ -642,33 +642,26 @@ protected Response sendRequest(RestRequest restRequest)
     private void logResponseInformation(RestRequest restRequest, Response returnedResponse)
     {
         String responseSizeString = returnedResponse.getHeader("Content-Length");
-        if (responseSizeString != null && Integer.valueOf(responseSizeString) > IGNORE_CONTENT_LIMIT_BYTES)
+        if (responseSizeString != null)
         {
-            LOG.info("On {} {}, received a response size that was {} bytes.\n"
-                    + "This is bigger than the limit of {} bytes so its content will not be displayed: \n", restRequest.getHttpMethod(),
-                restRequest.getPath(), Integer.valueOf(responseSizeString), IGNORE_CONTENT_LIMIT_BYTES);
-        }
-        else
-        {
-            if (returnedResponse.getContentType().contains("image/png"))
-            {
-                LOG.info("On {} {}, received the response with an image and headers: \n{}", restRequest.getHttpMethod(), restRequest.getPath(),
-                    returnedResponse.getHeaders().toString());
-            }
-            else if (returnedResponse.getContentType().contains("application/json") && !returnedResponse.asString().isEmpty())
-            {
-                LOG.info("On {} {}, received the following response \n{}", restRequest.getHttpMethod(), restRequest.getPath(),
-                    Utility.prettyPrintJsonString(returnedResponse.asString()));
-            }
-            else if (returnedResponse.getContentType().contains("application/xml") && !returnedResponse.asString().isEmpty())
-            {
-                String response = parseXML(returnedResponse);
-                LOG.info("On {} {}, received the following response \n{}", restRequest.getHttpMethod(), restRequest.getPath(), response);
-            }
-            else
-            {
-                LOG.info("On {} {}, received the following response \n{}", restRequest.getHttpMethod(), restRequest.getPath(),
-                    ToStringBuilder.reflectionToString(returnedResponse.asString(), ToStringStyle.MULTI_LINE_STYLE));
+            if (Integer.valueOf(responseSizeString) > IGNORE_CONTENT_LIMIT_BYTES) {
```

Comment:
Please revert your formatting changes here. It makes it very hard to review what you've changed.

Reply:
I have only added if (responseSizeString != null) condition at top level so I had to change indentation for inner code in the condition. Please let me know if indentation need to be reverted bcoz in that case it will be badly indented.

Reply:
Have a look at the braces - you've moved them all, which makes this hard to review.